www/front_src/src/components/form-fields/CheckboxField.js

```diff
@@ -32,18 +34,17 @@ const CheckboxField = ({
           onChange && callbackWithValue(trueValue, falseValue, onChange)
         }
         onBlur={onBlur && callbackWithValue(trueValue, falseValue, onBlur)}
-        className="custom-control-input"
+        className={styles["custom-control-input"]}
         type="checkbox"
       />
-      <label htmlFor={rest.id} class="custom-control-label">
+      <label htmlFor={rest.id} className={styles["custom-control-label"]}>
         {label}
         {info}
       </label>
     </div>
     {error ? (
-      <div class="invalid-feedback">
-        <i class="fas fa-exclamation-triangle" />
-        <div class="field__msg  field__msg--error">{error}</div>{" "}
+      <div className={styles["invalid-feedback"]}>
+        <div className={classnames(styles["field__msg"], styles["field__msg--error"])}>{error}</div>{" "}
```

Comment:
is the "exclamation-triangle" picture useless ?

Reply:
or is it already displayed using the "styles["field__msg--error"]" ?

Reply:
exclamation triangle did not work cause we didn't embed fontawesome